### PR TITLE
Fixing tarball build

### PR DESCRIPTION
**Pull request by sahilTakiar**
@ydai can you review?

* Fixes the tarball build
* Added the version to the name of the tarball

### CODE REVIEW

File: gobblin-distribution/build.gradle [hunk 1]
```diff
@@ -63,9 +58,8 @@ task build(type: Tar, overwrite: true) {
 
   doLast {
     copy {
-      from buildDir.path + '/distributions/' + distributionTarballName + '.tar.gz'
+      from buildDir.path + '/distributions/' + baseName + '-' + project.version + '.tar.gz'
```
Comment (ydai1124) [inline gobblin-distribution/build.gradle]: Do we need to change the wiki page as well?

**Comment by ydai1124** [comment]
One comment. LGTM.

File: gobblin-distribution/build.gradle (hunk 1 above)
Comment (sahilTakiar) [inline gobblin-distribution/build.gradle]: Updated the wiki pages

Comment (ydai1124) [inline gobblin-distribution/build.gradle]: LGTM.

**Pull request merged**